tests/integration/session_integration_multiple/msg.rflx

```diff
@@ -0,0 +1,9 @@
+package Msg is
+
+   type T is mod 2 ** 8;
+   type Message is
+      message
+         Field : T;
+      end message;
+
+end Msg;
```

Comment:
I don't see the cost of adding a new message type here, could you please explain?

Reply:
I see two things here:

- 1. It is an additional specification file which needs to be maintained in case we change the specification language.
- 2. Having a slightly different message type in each integration test makes it harder to understand/review the corresponding session specifications.

Both arguments are probably not that strong. But what would be the advantage of not just reusing an existing message specification?

Reply:
The direct cost of adding a single message is indeed negligible. However, the more separate message we add for test cases, the more expensive fundamental refactorings get (e.g. where you change the syntax and have to touch basically every single message). We know that as that's where we are coming from. Earlier, we just added new messages as we saw fit up to the point where you would want to avoid refacting as it was was too painful. It still is painful sometimes, but we should try not to worsen it (e.g. by using as much common test infrastructure as possible).

Reply:
I see. I guess this is because Recordflux is still very young so we can expect many changes to the input language. I do think in the longer term having just one message for all integration/feature tests weakens those tests quite a bit. I will change to use universal.rflx.

Reply:
Fixed in 2e2523e1e467e10620a0e2875b7c58b968e77da9

Reply:
> I do think in the longer term having just one message for all integration/feature tests weakens those tests quite a bit.

Intentionally adding new message formats, which have a fundamental difference, is a good thing. But I don't see how randomly adding simple message specifications are beneficial in the long term.